src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__box calendar__box--length--29 calendar__box--start-day--sun">
```

Comment:
I guess you dont need this box element, like you could easily use all of this styles and modifiers for your `calendar ` block)

Reply:
Agree with @MaxManis